pylint/checkers/refactoring.py

```diff
@@ -476,8 +478,118 @@ def visit_unaryop(self, node):
                              args=(node.as_string(), suggestion))
 
 
+def _is_len_call(node):
+    """Checks if node is len(SOMETHING)."""
+    return (isinstance(node, astroid.Call) and isinstance(node.func, astroid.Name) and
+            node.func.name == 'len')
+
+def _is_constant_zero(node):
+    return isinstance(node, astroid.Const) and node.value == 0
+
+def _node_is_test_condition(node):
+    # the node is an if, while, assert of if expression statement
+    return isinstance(node, (astroid.If, astroid.While, astroid.Assert, astroid.IfExp))
+
+
+class LenChecker(checkers.BaseChecker):
+    """Checks for incorrect usage of len() inside conditions. pep8 states:
```

Comment:
Super small nit:  Move `pep8 states:`... to next line to clean up one-line docstring for this method.  Also, `pep8` should be capitalized.